next.clayui.com/content/docs/css/components/popovers-and-tooltips.md

```diff
@@ -0,0 +1,469 @@
+---
+title: "Popovers and Tooltips"
+description: "Popovers and Tooltips are patterns used to show helpful information in two different ways."
+---
+
+<div class="nav-toc">
+
+- [Popover](#popover)
+    - [Position](#popover-position)
+        - [Top](#popover-top)
+        - [Right](#popover-right)
+        - [Bottom](#popover-bottom)
+        - [Left](#popover-left)
+- [Tooltip](#tooltip)
+    - [Position](#popover-position)
```

Comment:
should be `#tooltip-position`

Reply:
Actually I don't think any of these work since the headers don't contain the `popover` or `tooltip` portions. You may actually have to have the prefix, unless markdown provides a syntax for custom `id` for headers.